.github/workflows/build-test.yaml

```diff
@@ -0,0 +1,62 @@
+name: Build/Test
+
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  validate-protobuf:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          path: ./
+      -
+        name: Install protoc
+        uses: arduino/setup-protoc@v3
+        with:
+          version: '25.3'
+      -
+        name: Install just
+        uses: extractions/setup-just@v1
```

Comment:
I'm not sure the UI will make any use of protobuf, so we may be able to drop this? I'm out of touch with this repo, so I could very well be wrong. Either we'll drop an extra job, or I'll learn something =)